packages/react-accordion/src/components/AccordionHeader/useAccordionHeader.ts

```diff
@@ -51,6 +51,14 @@ export const useAccordionHeader = (
   const size = useContextSelector(AccordionContext, ctx => ctx.size);
   const id = useId('accordion-header-', props.id);
   const panel = useDescendants(accordionItemDescendantContext)[1] as AccordionItemDescendant | undefined;
+  const handleKeyDown = React.useCallback(
```

Comment:
```suggestion
  const handleKeyDown = useEventCallback(
```

If `onAccordionHeaderClick` comes from user and is not stable, we should use `useEventCallback`.

---

However, I don't see any need to memoize this as it goes to a slot that it is represented by native primitive.